### ISSUE TITLE
Disable Inheritance for Automatic Alt Tag Assignment

### ISSUE THREAD

**Issue by looserben** [shopware/platform#16912]
### Shopware Version

6.7.10.1

### Affected area / extension

Platform(Default)

### Actual behaviour

If automatic alt tag generation is enabled for a specific language, the tags are automatically applied to all other language versions of the image as well. This poses a problem from an SEO perspective, as it results in tags from incorrect languages ​​appearing within mismatched translations. 

### Expected behaviour

This automatic inheritance feature should be disabled to ensure that alt text is populated only within the selected translation.

### How to reproduce

.

**Comment by dneustadt** [comment]
@looserben The issue here isn't exactly straightforward. The "alt tag" field is a translatable field; these fields always involve a mechanism where the one default language value is inherited by other languages if no translation is available for them specifically.

The change that was implemented previously prevented values ​​from being generated for the default language in every instance, even when that wasn't the target language.

However, that doesn't alter the general inheritance behavior, which is deeply rooted in Shopware. So, once values ​​for the default language have been generated, they continue to be inherited as long as the other languages ​​don't have their own specific values. Changing this behavior would essentially constitute a breaking change.

The only clean solution I see here is to generate specific alt text values ​​for all the other languages ​​being used.

**Comment by dneustadt** [comment]
@looserben I'm going to close this issue temporarily. As already mentioned this is not a bug in the extension but a core feature where translated fields are inherited from the default language. This way, if you wanted one shared `alt` attribute for all languages, you could set it for the default language and leave it blank for other languages. If this is not desired, values for `alt` attribute should then be generated for all languages, not just for the default language, so each language holds their own value for the `alt` attribute.

There was an issue where the values were always generated for the default language, even if it was not the targeted language. That issue was solved with https://github.com/shopware/SwagCommercial/pull/2802
